lib/src/alert_dialog/show_alert_dialog.dart

```diff
@@ -62,12 +62,16 @@ Future<T?> showAlertDialog<T>({
             barrierDismissible: barrierDismissible,
           ),
           builder: (context) => AlertDialog(
+            backgroundColor: theme.cardColor,
```

Comment:
Shouldn't be set `theme.cardColor` because it breaks default color(`dialogTheme.backgroundColor ?? Theme.of(context).dialogBackgroundColor`).
https://github.com/flutter/flutter/blob/a603714610f8c9c831e7d85071274460d70c18da/packages/flutter/lib/src/material/dialog.dart#L145